src/main/java/com/ably/kafka/connect/utils/StructUtils.java

```diff
@@ -0,0 +1,74 @@
+package com.ably.kafka.connect.utils;
+
+import com.google.gson.Gson;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Struct;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class StructUtils {
+    public static String toJsonString(final Struct struct, Gson gson) {
+        return gson.toJson(structMap(struct));
+    }
+
+    private static Map<String, Object> structMap(final Struct struct) {
+        final Map<String, Object> dataMap = new LinkedHashMap<>(struct.schema().fields().size());
+
+        for (Field field : struct.schema().fields()) {
```

Comment:
`field` can be `final` here.

```suggestion
        for (final Field field : struct.schema().fields()) {
```

Reply:
00c7e52fed855fdf3d9f266f79eb400978dc820e